README.md

```diff
@@ -75,13 +75,13 @@ id="sanctuary-types-return">[1](#sanctuary-types)</sup>
     - Type variables can take any type unless they have been restricted by
       means of type constraints (see fat arrow below).
 * `->` (arrow) _Function type constructor._
-    - `->` is an _infix_ type constructor that takes two type arguments and
-      gives the type of functions between these two type arguments.
+    - `->` is an _infix_ type constructor that takes two type arguments where
+      left argument is the input type and the right argument is the output type.
     - `->`'s input type can be a grouping of types to create the type of a
-      function which accepts more than one argument (i.e. has arity greater
-      than one). When used in this way, the syntax is: `(<input-types>) ->
-      <output-type>`, where `<input-types>` comprises two or more comma–space
-      (`, `)-separated type representations.
+      function which accepts zero or more arguments. The syntax is:
+      `(<input-types>) -> <output-type>`, where `<input-types>`comprises zero
+      or more comma–space (`, `)-separated type representations and parens may
```

Comment:
I don't know how to put no-break space for markdown. Does the same apply to these expressions?

> 1. `t(u.traverse(F, x => x))` is equivalent to `u.traverse(G, t)` for any

> 3. `u.traverse(Compose, x => new Compose(x))` is equivalent to

> `new Compose(u.traverse(F, x => x).map(x => x.traverse(G, x => x)))

> `M.chainRec((next, done, v) => p(v) ? d(v).map(done) : n(v).map(next), i)`

> 3. `A.of(y).ap(u)` is equivalent to `u.ap(A.of(f => f(y)))` (interchange)